### PR TITLE
fix: better handling of cases where no data is found

### PR DESCRIPTION
**Pull request by ffsinger**
There were errors on the twitter experience when the user inputs an advertiser that doesn't exist in the data. Now the page shows "No relevant data found".

### CODE REVIEW

**Review by streitlua — APPROVED** [review]
Looks good to me, although the problem of the results not refreshing when we change the query still exists.

**Comment by ffsinger** [comment]
> the problem of the results not refreshing when we change the query still exists.

It has been fixed in #234 and when I test it works. Are you sure ? How does it happen ?

File: components/unit/UnitQuery.vue [hunk 1]
```diff
@@ -115,7 +115,7 @@
           </template>
           <template v-else>
             <VRow>
-              <VCol>
+              <VCol style="text-align: center">
```
Comment (andreaskundig) [inline components/unit/UnitQuery.vue:118]: It's better if we can avoid using inline styles, they make it harder to have a consistent look everywhere. In this case, you should be able to use [justify from the vuetify grid system](https://vuetifyjs.com/en/components/grids/#justify)

**Review by andreaskundig — CHANGES_REQUESTED** [review]
Otherwise I trust Luà's assessment that everything's ok

**Comment by streitlua** [comment]
I found another issue when testing with the `twitter.zip` archive (although this is likely related to the sql pipeline and not this PR):
image
image

**Comment by streitlua** [comment]
> > the problem of the results not refreshing when we change the query still exists.
> 
> It has been fixed in #234 and when I test it works. Are you sure ? How does it happen ?

You are right, it is working. I was testing random words and then `apple` but was still seeing no results. The search is case-sensitive (maybe it shouldn't be?) so I can see results when searching `Apple`.

File: components/unit/UnitQuery.vue (hunk 1 above)
Comment (ffsinger) [reply components/unit/UnitQuery.vue:118]: If I use something like this:

```
<VRow justify="center">
  <VCol cols="4">
    <p><i>No relevant data found</i></p>
  </VCol>
</VRow>
```

It centers the column but not the text inside it:

Screenshot_20211124_094415

I would say that it doesn't look as good as having the text fully centered, what do you think ? Can we achieve this without using `style="text-align: center"` ?

Comment (andreaskundig) [reply components/unit/UnitQuery.vue:118]: It looks fine to me. I don't like centered text. 

And the text in the description is also left-justified (could be something I did, actually).

Comment (andreaskundig) [reply components/unit/UnitQuery.vue:118]: text-center is also an acceptable solution

**Comment by ffsinger** [comment]
It would indeed be better if it wasn't case-sensitive, I'll see if I can fix this

**Comment by ffsinger** [comment]
Case-sensitivity has been fixed in commit 557b004a13732b86bd5e532405b96e468fcedc68

**Pull request merged**